### DISCUSSION TITLE
Installer crashing while trying to update/install (Win10 1809)

### DISCUSSION THREAD

**Discussion by ghost** [Difegue/LANraragi#522] [Q&A] [answered]
**LRR Version and OS**
Was installed 0.7.7, trying to update to  0.8
Windows 10 LTSC 1809 Build 17763.1817

**Bug Details**
Installer stops while trying to update with error: The installer has stopped prematurely.
I was running 0.7.7 fine with a fix posted here to use network disks.

**Matching Logs**
PS C:\Users\wolf\Downloads> wslconfig.exe /l
Windows Subsystem for Linux has no installed distributions.
Distributions can be installed by visiting the Microsoft Store:
https://aka.ms/wslstore

**Screenshots**
![image](https://user-images.githubusercontent.com/92317622/136804590-ca7896ff-54a4-4e8f-aa28-d3a8459f1763.png)

[install.log](https://github.com/Difegue/LANraragi/files/7323058/install.log)
.

**Comment by Difegue** [comment, marked as answer]
Support for Windows 10 1809 has been dropped in 0.8.0, as mentioned in the [Release Notes.](https://github.com/Difegue/LANraragi/releases/tag/v.0.8.0)  

However, you can keep using your current version of the Karen bootstrapper and update/register the WSL distro manually as a workaround (quick copy-paste from discord, excuse the poor formatting):  

> You can technically keep using new LRR versions by registering the WSL distro manually.
The package.tar is always located in the install directory and as long as you register that with the name "lanraragi", the bootstrapper will pick it up
You'll probably need https://github.com/DDoSolitary/LxRunOffline , since I don't think 1809 has an easy built-in registration command  
I'm aware 1809 is LTSC and some people stay on it for reasons, but the old install workflow was kinda kludgy.
